src/main/java/com/odysseusinc/arachne/portal/model/ExternalLogin.java

```diff
@@ -0,0 +1,105 @@
+/*
+ *
+ * Copyright 2022 Odysseus Data Services, inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Created: February 04, 2022
+ *
+ */
+package com.odysseusinc.arachne.portal.model;
+
+import java.time.Instant;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "external_logins")
+public class ExternalLogin {
+    @Id
+    @SequenceGenerator(name = "external_logins_pk_sequence", sequenceName = "external_logins_id_seq", allocationSize = 1)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "external_logins_pk_sequence")
+    protected Long id;
+
+    @Column(nullable = false, name = "provider")
+    protected String provider;
+
+    @Column(nullable = false, name = "sub")
+    protected String sub;
+
+    @Column(name = "email")
+    protected String email;
+
+    @Column(name = "created")
+    private Instant created;
+
+    @ManyToOne(targetEntity = User.class, cascade = CascadeType.PERSIST)
```

Comment:
do you really need CascadeType.PERSIST here?

Reply:
For the moment - probably no.